website/docs/r/bot_channel_directline.html.markdown

```diff
@@ -58,14 +58,24 @@ A `site` block has the following properties:
 
 * `name` - (Required) The name of the site
 
+* `application_id` - (Optional) The application ID for this site.
+
+* `block_user_upload_enabled` - (Optional) Is the block user upload enabled for this site? Defaults to `false`.
+
 * `enabled` - (Optional) Enables/Disables this site. Enabled by default Defaults to `true`.
 
+* `endpoint_parameters_enabled` - (Optional) Is the endpoint parameters enabled for this site?
+
+* `no_storage_enabled` - (Optional) Is no storage enabled for this site?
+
 * `v1_allowed` - (Optional) Enables v1 of the Directline protocol for this site. Enabled by default Defaults to `true`.
 
 * `v3_allowed` - (Optional) Enables v3 of the Directline protocol for this site. Enabled by default Defaults to `true`.
 
 * `enhanced_authentication_enabled` - (Optional) Enables additional security measures for this site, see [Enhanced Directline Authentication Features](https://blog.botframework.com/2018/09/25/enhanced-direct-line-authentication-features). Disabled by default.
 
+* `tenant_id` - (Optional) The Tenant ID for this site.
```

Comment:
Does the API support cross tenant scenario? If not, probably we don't need to expose this `tenant_id` as we already have that information in the provider? Otherwise, we shall mention it in the document.

Reply:
Service team confirmed that it will be deprecated in next version. So I removed it.